charts/kong/values.yaml

```diff
@@ -347,6 +347,7 @@ migrations:
   # This is helpful in certain non-Helm installation situations such as GitOps
   # where additional control is required around this job creation.
   jobAnnotations: {}
+  backoffLimit:
```

Comment:
It seems we should add some comments here, as we do for some of the surrounding values, which links to the relevant documentation and/or has some minimal explanation of the usage?

Reply:
Hurray forgetting to save. f8b678c45